scripts/lib/setup_venv.py

```diff
@@ -35,6 +35,10 @@
     # on upgrade of a production server, and it's not worth adding
     # another call to `apt install` for.
     "jq",                   # Used by scripts/lib/install-node to check yarn version
+
+    # Fixes missing sasl.h headers when
+    # compiling python-ldap from source.
+    "libsasl2-dev",
```

Comment:
This probably deserves a comment similar to the existing ones above.  And @andersk should we rework the comments above to have the comment on the line before the package?  That will be a useful refactoring in preparation for Black.

Reply:
(Also, we should look up when `python-ldap` started requiring that package, since that'll determine if we need to backport this)

Reply:
Black is fine with comments in either position. It won’t align the right comments in a column, but it’s not like we’ve been particularly competent at that; all that ever happens when you try to make pretty columns is they get destroyed by the next committer.

`# For building python-ldap from source` would be a totally sufficient comment, and it might as well be formatted like the others.

Reply:
Done.